core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.ts

```diff
@@ -123,6 +127,11 @@ angular.module('oppia').component('translationOpportunities', {
             getPresentableOpportunitiesData);
       };
 
+      ctrl.resetCursorStyle = function() {
+        document.body.style.cursor = '';
+      };
+
+
```

Comment:
Remove extra line.

Reply:
My recent commit should fix this. The functions will be equispaced now. Thanks!